recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+import yaml
+
+
+
+class Open62541Conan(ConanFile):
+    name = "open62541"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Open source implementation of OPC UA (OPC Unified Architecture) aka IEC 62541 licensed under Mozilla Public License v2.0"
+    topics = ("conan", "opcua", "iec62541")
+    homepage = "http://open62541.org"
+    license = "MPL-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    exports = "submoduledata.yml"
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "multithread": "ANY",                                                   # Level of multithreading (0-99: No Multithreading, 100-199: Thread-Safe API, >=200: Internal Threads
+                                                                                # multithread configuration may be
+                                                                                #       1.0 : False, True
+                                                                                #       1.1 : False, thread-safe, internal-threads or an integer
+        "tls": [False, "openssl", "mbedtls-apache", "mbedtls-gpl"],
```

Comment:
```suggestion
        "tls": [False, "openssl", "mbedtls"],
```

Eventually create a `auto` option:
```python
        "tls": [False, "auto", "openssl", "mbedtls"],
...
See the cmake recipe how to use this:
https://github.com/conan-io/conan-center-index/blob/master/recipes/cmake/3.x.x/conanfile.py#L17